src/robot/conf/languages.py

```diff
@@ -305,6 +307,7 @@ class En(Language):
     library_setting = 'Library'
     resource_setting = 'Resource'
     variables_setting = 'Variables'
+    name_setting = "Name"
```

Comment:
For consistency single quotes would be better than double quotes. Same also with other `name_settings`.

Reply:
switched to single quotes